metricbeat/module/ceph/docker-compose.yml

```diff
@@ -13,11 +13,11 @@ services:
       - 8003
       - 8080
   ceph-api:
-    image: docker.elastic.co/integrations-ci/beats-ceph:master-6373c6a-jewel-centos-7-x86_64-1
+    image: docker.elastic.co/integrations-ci/beats-ceph:${CEPH_VERSION:-master-97985eb-nautilus-centos-7-x86_64}-1
```

Comment:
I'm not sure why this image is version `-1` and the regular ceph is `-2`. @jsoriano could you please 👀 here?

Reply:
I think we wanted to keep using an specific version here, that is different to the one in the `ceph` service.